content/en/docs/demos/f5_nginx_ingress_controller.md

```diff
@@ -0,0 +1,420 @@
+## This document outlines how to integrate F5 NGINX Ingress Controller with Open Service Mesh (OSM)
+
+Open Service Mesh will work with both versions of [F5 NGINX Ingress controller](https://github.com/nginxinc/kubernetes-ingress): the free as well as the NGINX Plus versions.
+
+Below is a link to the official F5 NGINX Ingress controller documentation.    
+[F5 NGINX Ingress controller](https://docs.nginx.com/nginx-ingress-controller/)
+
+# Integrating NGINX Ingress Controller with Open Service Mesh
+
+There are two ways to integrate the NGINX Ingress Controller with Open Service Mesh (OSM):
+
+1. Injecting an envoy sidecar directly with NGINX Ingress Controller.
+2. Using the Open Service Mesh `ingressBackend` "proxy" feature.
+
+# First step is to install F5 NGINX Ingress controller. Here the links to the install guides:
+
+[Using Helm to install NGINX Ingress](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-helm/)
+[Using Manifests to install NGINX Ingress](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/)
+
+
+When using the sidecar method, ensure that you add the correct annotations listed below. This ensures proper integration of NGINX Ingress Controller with the envoy sidecar proxy.
+At a minimum, the following annotation needs to be set on your deployment:
+
+```yaml
+annotations:
+  openservicemesh.io/inbound-port-exclusion-list: "80,443"
+````
+
+
+# NGINX Ingress controller and OSM with sidecar injected 
+
+Install OSM in the cluster
+
+```bash
+osm install --mesh-name osm-nginx --osm-namespace osm-system
+```
+
+### Mark F5 NGINX Ingress controller namespace for sidecar injection
+
+*NOTE:* Depending on how you install NGINX Ingress controller, you might need to create the `namespae`. For example, if you are using manifests to install NGINX Ingress controller, you can complete all of the steps on our documentation page, *EXCEPT*, actually deploying NGINX Ingress controller. This is because, when using the sidecar approach, OSM needs to "manage" the namespace so it knows what `namespaces` it needs to inject sidecars into.
```

Comment:
```suggestion
*NOTE:* Depending on how you install NGINX Ingress Controller, you might need to create the `namespace`. For example, if you are using manifests to install NGINX Ingress Controller, you can complete all of the steps on our documentation page, *EXCEPT*, actually deploying NGINX Ingress Controller. This is because, when using the sidecar approach, OSM needs to "manage" the namespace so it knows what `namespaces` it needs to inject sidecars into.
```